presto-cache/src/main/java/com/facebook/presto/cache/alluxio/EvictionPolicy.java

```diff
@@ -16,8 +16,8 @@
 public enum EvictionPolicy
 {
     FIFO("alluxio.client.file.cache.evictor.FIFOEvictor"),
```

Comment:
Do we need to change this as well? I see others all have "Cache" 😂

Reply:
this one is really named as `FIFOEvictor` , that's why I thought others are named in the same way. My bad